README.md

```diff
@@ -30,12 +32,34 @@ export default {
 new NpmAutoInstallWebpackPlugin(options)
 ```
 
-### options.autoInstall
+### options.install
 
 Auto install missing dependencies with `npm install` command.
 
 * Default: `true`
 
+### options.strategy
```

Comment:
This makes me thinking: Why not add a `options.useYarn` instead? Then we can simply do the same thing without configuring `strategy` for now.